### ISSUE TITLE
Will it be possible to add other types of airfoil definitions instead of point clouds (e.g. CST definitions)?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ebouhoubeiny/cpacs#225]
```
Occurs in xPath: 
/cpacs/vehicles/profiles/wingairfoils

Problem/Question: 
Will it be possible to add other types of airfoil definitions instead of point 
clouds (e.g. CST definitions)?

Reply: 
Yes, CPACS 2.1

daniel.boehnke@dlr.de
```

Original issue reported on code.google.com by `daniel.b...@googlemail.com` on 25 Jan 2012 at 10:56

**Comment by GoogleCodeExporter** [comment]
Original comment by `daniel.b...@googlemail.com` on 1 Feb 2012 at 1:40
- Added labels: **version-CPACS2.1**

**Comment by GoogleCodeExporter** [comment]
Original comment by `daniel.b...@googlemail.com` on 1 Feb 2012 at 1:41
- Added labels: **milestone-CPACS2.1**

**Comment by GoogleCodeExporter** [comment]
Original comment by `daniel.b...@googlemail.com` on 8 Feb 2012 at 9:46

**Comment by GoogleCodeExporter** [comment]
```
The proposed schema for the CST definition of wingAirfoils and fuselageProfiles 
is

uID
n1  - double
n2  - double
coefficientsUpper - doubleVector
coefficientsLower - doubleVector
scalingFactor  - double
```

Original comment by `daniel.b...@googlemail.com` on 21 Mar 2013 at 9:34

**Comment by GoogleCodeExporter** [comment]
```
Included in CPACS 2.1, see attached powerpoint slides
```

Original comment by `daniel.b...@googlemail.com` on 16 Apr 2013 at 8:41

Attachments:
- [2013 03 - CST for CPACS.pptx](https://storage.googleapis.com/google-code-attachments/cpacs/issue-225/comment-5/2013 03 - CST for CPACS.pptx)

**Comment by GoogleCodeExporter** [comment]
Original comment by `daniel.b...@googlemail.com` on 16 Apr 2013 at 8:51
- Changed state: **Done**
